### ISSUE TITLE
audio translations getting cut off

### ISSUE THREAD

**Issue by dimm0k** [ankidroid/Anki-Android#4541]
###### Research
*Enter an [ x ] character to confirm the points below:*

[ x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum

###### Reproduction Steps

1. card answers that use TTS get cut off in the beginning, specifically Japanese
2. 
3. 

###### Expected Result

full speech of the text for the resulting answer in Japanese

###### Actual Result

partial cut of the beginning portion of the speech and if the answer is short enough, you won't even hear the speech at all. replaying the answer however plays it in full.

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

AnkiDroid Version = 2.8beta3
Android Version = 7.0
ACRA UUID = 0f5d7e80-8eb1-4be1-9e8e-295ba460fdb4

**Comment by hssm** [comment]
Go to your device's settings at `Settings -> Language & input -> Text-to-speech output`.

If you're using the Samsung TTS engine, change it to the Google one.

**Comment by dimm0k** [comment]
I have the LGV20... the only TTS engine listed and already selected is the Google one

**Comment by rlpowell** [comment]
This almost certainly has nothing to do with ankidroid; I get this with all sorts of apps.

In my experience, this only happens with bluetooth headsets, and can be fixed by changing bluetooth headsets, but I suspect it could happen with wired headsets as well.  My best guess is that it has to do with the headset deciding that since no sound is playing, it can turn audio off until sound plays.

I have fixed it by buying a new headset.

I have also fixed it by playing a very very quiet music track in the background, to keep the audio going.
